app/views/idv/shared/_document_capture.html.erb

```diff
@@ -43,6 +43,7 @@
       how_to_verify_url: idv_how_to_verify_url,
       previous_step_url: @previous_step_url,
       locations_url: idv_in_person_usps_locations_url,
+      address_search_url: '',
```

Comment:
wait is this missing an actual URL?
```suggestion
      address_search_url: idv_in_person_address_url,
```

Reply:
No, the PR description has more info but short version is that there is no current valid URL to use for this.

Reply:
ah whoops, I missed that. thx!